### DISCUSSION TITLE
What to do with the Tutorial Seminar Series videos?

### DISCUSSION THREAD

**Discussion by jukent** [ProjectPythia/pythia-foundations#118] [General]
There were a lot of good ideas around how to best celebrate and utilize these tutorial recordings during today's meeting. As promised I have compiled all of the tutorial videos, their content locations, and the blog posts (and sometimes a FAQ too) into this [Google Sheet](https://docs.google.com/spreadsheets/d/1fsIYztL4hkG6QXkIjos3_K_-Wa5wF8EKssLBw1cXChU/edit?usp=sharing).

**Comment by clyne** [comment]
Thanks for putting this together, @jukent. It looks like there is a good chance that many of the notebooks could be consolidated into a single repo or two, and that would help with maintenance. 

I think that there are two issues we need to contend with that we didn't really distinguish today: 1. where and how to maintain the notebooks, and 2. how to present the content to the users in a way that they can navigate and find what they want. I have a proposal to get the conversation started.

1. Though it would be nice to gather all of the notebooks in a single location (repo) that might not be practical given different environment needs. I suggest we do what we can to collect as many notebooks into as few repos under github.com/ProjectPythia as is reasonably possible. Perhaps future versions of the Jupyter technologies will allow for supporting different configurations within a single repo. Moving forward we can do what we can to encourage tutorial creators to use an existing repo/environment when possible. 

2. We've already made a substantial investment in the external links gallery. Why not simply repurpose it for a "Pythia Tutorial Collection"?  There are a number of attractions for doing this:

- It looks professional and is already familiar to Pythia users
- Minimal staff costs for development
- The "cards" can easily contain all of the relevant information for each tutorial, and provide a link to the video and the binderized notebook.
- There is a filtering capability to facilitate finding different kinds of content.
- We can repurpose the issue template to simplify submission of new tutorials. We may not want to limit the collection to Pythia "owned" content.
- For each of  the cards we could provide some kind of versioning information for the video and the notebook. The further apart the two, the less likely they match. 

Just wanted to get the ball rolling with a plan that would address the basic needs and require minimal staff investment. We can always refactor later :-). Let me know your thoughts, everyone!

**Reply by jukent** [reply]
Great thoughts @clyne Do you mean we should create a new gallery card for every tutorial seminar recording, and it will have a unique tag so we can search and get a gallery of only the tutorial seminar recordings -- OR do you mean we should copy most of the code to have a separate gallery that is only for the tutorial seminar recordings? I can start working on either one of these (after I finish the Jupyter content).

**Reply by clyne** [reply]
I guess I was thinking that latter: having a completely separate gallery for the training series, but reusing the infrastructure developed for the external links gallery to support it.  What do others think?

**Comment by brian-rose** [comment]
From discussion at the IWG today, this becomes straightforward once we merge https://github.com/ProjectPythia/projectpythia.github.io/pull/160 (affiliation tags) and find a solution for https://github.com/ProjectPythia/projectpythia.github.io/issues/125 (multiple gallery filters). The user just needs to apply "Pythia" and "Video" or "Tutorial" filters (or whatever we choose) and get a dynamically rendered gallery of these videos.

We could then have a direct link to the gallery with these filters pre-applied.
